pkg/insights/resyncer.go

```diff
@@ -80,6 +99,7 @@ func NewResyncer(config *Config) component.Component {
 }
 
 func (r *resyncer) Start(stop <-chan struct{}) error {
+	r.info = map[model.ResourceKey]syncInfo{} // clear the map if component is restarted
```

Comment:
It seems like `r.info` map is accessed from different goroutines, probably some locks are required

Reply:
fair point